### PR TITLE
Fix missing source strings and inconsistent lead text style in admin UI

### PR DESCRIPTION
**Pull request by Gargron**

### CODE REVIEW

**Review by ykzts — APPROVED** [review]

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -91,8 +91,6 @@ def visibility_icon(status)
       fa_icon('unlock', title: I18n.t('statuses.visibilities.unlisted'))
     elsif status.private_visibility? || status.limited_visibility?
       fa_icon('lock', title: I18n.t('statuses.visibilities.private'))
-    elsif status.direct_visibility?
-      fa_icon('envelope', title: I18n.t('statuses.visibilities.direct'))
     end
```
Comment (ClearlyClaire) [inline app/helpers/application_helper.rb:94]: why this change…?

Comment (Gargron) [reply app/helpers/application_helper.rb:94]: Ah, maybe a mistake. I thought we didn't render direct statuses in the public view. For some reason we don't have `statuses.visibilities.direct` in our strings.

Comment (ClearlyClaire) [reply app/helpers/application_helper.rb:94]: Ah yes, I had missed the translation was missing :/

The proper fix should probably to add that translation string and add back the icon, even if people do not commonly access DMs through this interface, it would be reassuring to them to see a clear indication that they are indeed DMs
